.prettierignore

```diff
@@ -0,0 +1,6 @@
+*.min.js
+node_modules/
+flow-typed/
+cypress/
```

Comment:
I wonder if we should limit this to ignoring things that are actually used by this project? Or at least things that are _likely_ to be used by this project. I doubt `cypress` or `lib` will ever be used here. Perhaps `flow-typed` will someday, so maybe that can stay. But even that could just be added with whatever PR would add it.

Reply:
@jdeal Right that makes sense -- WRT to `lib`, that represents the directory that the module is built into, I think it should stay